Libraries/Boards/MAX32655/EvKit_V1/examples.txt

```diff
@@ -5,28 +5,37 @@ BLE5_ctr
 BLE_datc
 BLE_dats
 BLE_fcc
-BLE_FreeRTOS
 BLE_fit
+BLE_fit_FreeRTOS
+BLE_FreeRTOS
 BLE_mcs
 BLE_otac
 BLE_otas
 BLE_periph
 Bootloader
 CRC
 DMA
+Dual_core_sync
+    Dual_core_sync_arm
+    Dual_core_sync_riscv
```

Comment:
In order for the project wizard to locate examples in subdirectories (like the AI85 CNN examples), you need to attach the path to the example name in one line. So these three lines should be replaced with:
`Dual_core_sync/Dual_core_sync_arm`
`Dual_core_sync/Dual_core_sync_riscv`